invenio_search/__init__.py

```diff
@@ -287,13 +287,13 @@ def index():
 
 Elasticsearch version support
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Because of the breaking changes that are introduced in Elasticsearch between
-major versions in relation to mappings, a specific directory structure has to
-be followed, in order to specify which JSON mapping files will be used for
-creating the Elasticsearch indices. For backwards compatibility with existing
-Invenio modules and installations, for Elasticsearch 2, mappings will be loaded
-from the root level of the package directory. You can see a full example in the
-``examples/data`` directory of the Invenio-Search repository:
+Major versions of Elasticsearch can include breaking changes to mappings so
+mappings for each version of Elasticsearch are stored in separate folders.
+Invenio-Search will pass these mappings to Elasticsearch when creating the
+indices. For backwards compatibility with existing Invenio modules and
+installations, Elasticsearch 2 mappings will be loaded from the root level of
```

Comment:
Here  `Elasticsearch 2` is mentioned while in https://github.com/inveniosoftware/invenio-search/pull/191/files#diff-26dee18463b5da190f343abcc74cf708 ES 2 and ES 5 lines are dopped.

Reply:
Discussed it and kept the note about ES2 since it's not completely removed yet but we still want to encourage users to use v6 and v7 since v2 and v5 will be deprecated soon.